package.json

```diff
@@ -23,6 +23,7 @@
     "gulp-eslint": "^3.0.1",
```

Comment:
Since this PR is only concerned with removing submission app, the changes in package.json is not required. So the idea is whenever we do any commit or submit any PR, we only touch or edit those files which are just concerned.

One of the reasons can be that you may have committed this file by mistake, so when ever commit, make sure to do `git status` or `git diff --staged`, so that you can properly track what all changes are you committing.